pom.xml

```diff
@@ -5,23 +5,25 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>3.2</version>
+    <version>3.28</version>
     <relativePath />
   </parent>
 
   <artifactId>credentials-binding</artifactId>
-  <version>1.18-SNAPSHOT</version>
+  <version>${revision}${changelist}</version>
   <packaging>hpi</packaging>
 
   <name>Credentials Binding Plugin</name>
   <description>Allows credentials to be bound to environment variables for use from miscellaneous build steps.
   </description>
   <url>https://wiki.jenkins.io/display/JENKINS/Credentials+Binding+Plugin</url>
   <properties>
-    <jenkins.version>2.7.3</jenkins.version>
-    <java.level>7</java.level>
+    <revision>1.18</revision>
+    <changelist>-SNAPSHOT</changelist>
+    <jenkins.version>2.60.3</jenkins.version>
```

Comment:
I know this is the baseline for Java 8, but do you think it's time to start relying on more recent LTS versions?

Reply:
We could; there is no particular need to. Here I am just matching the baseline selected in the upstream PR, which _was_ to be able to use `-source 8`.